src/main/java/org/snomed/snowstorm/core/data/services/BranchReviewService.java

```diff
@@ -228,6 +231,106 @@ public void applyMergeReview(MergeReview mergeReview) throws ServiceException {
 		branchMergeService.mergeBranchSync(mergeReview.getSourcePath(), mergeReview.getTargetPath(), concepts);
 	}
 
+	private Concept upgradeComponents(Concept sourceVersion, Concept targetVersion) {
```

Comment:
Essentially, this method copies properties from the latest Component (either Source or Target version). If International has been upgraded, then the Source version will be used.